README.md

```diff
@@ -1535,4 +1543,4 @@ My name is [Yakiv](https://github.com/Iakow) and I am a future kottan. Glad to b
 My name is [Stas](https://github.com/Jigsaff)
 Good luck to everyone!
 
----
\ No newline at end of file
+---
```

Comment:
It shouldn't be here too.

Reply:
Can you check all correct mistakes which i did?